docs/installations/installation-on-gnu-linux.md

```diff
@@ -0,0 +1,106 @@
+Installation on GNU/Linux
+=========================
+
+## Install via Conda
+
+Use conda or bioconda to install OpenMS.
+
+1. Follow the instructions to [install conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html).
+2. Install OpenMS using conda:
+   `conda install -c openms openms`
+3. Other OpenMS packages can be installed using:
+   ```
+   conda install -c openms pyopenms
+   conda install -c openms openms-thirdparty
+   conda install -c openms libopenms
```

Comment:
it would be nice to have a short summary of what these packages contain, but that can also be added later.
openms contains the C++ tools, libopenms is the C++ library and required for the C++ tools to work.
pyopenms is the python package that allows to use algorithms from libopenms in python.
openms-thirdparty are external tools that are wrapped in OpenMS with adapters and required if one wants to use the adapters in the openms package.